zebra/interface.c

```diff
@@ -2821,24 +2821,16 @@ DEFUN (no_shutdown_if,
 	return CMD_SUCCESS;
 }
 
-DEFUN (bandwidth_if,
-       bandwidth_if_cmd,
-       "bandwidth (1-100000)",
-       "Set bandwidth informational parameter\n"
-       "Bandwidth in megabits\n")
+DEFUN(bandwidth_if, bandwidth_if_cmd, "bandwidth (1-1000000)",
```

Comment:
Please no whitespace changes here.

Reply:
frrbot will not be happy. but ok if you want

Reply:
As I've explained countless times, defuns are not subject to the style guide.

Reply:
did not everyone complain about how to avoid DEFUN from appear on frrbot ?

Reply:
Yes, several people expressed that they wanted frrbot to handle `DEFUN`s, and if memory serves I said I haven't found a way of doing it; frrbot uses `clang-format` and `DEFUN`s would need to be wrapped in some markup that makes `clang-format` ignore it.

Does your comment have something to do with this code review?